assets/src/blocks/Splittwocolumns/SplittwocolumnsEditor.js

```diff
@@ -0,0 +1,57 @@
+import { Fragment } from '@wordpress/element';
+import { BLOCK_NAME, VERSION } from './SplittwocolumnsBlock';
+import { SplittwocolumnsFrontend as Frontend } from './SplittwocolumnsFrontend';
+import { SplittwocolumnsSettings as SidebarSettings } from './SplittwocolumnsSettings';
+import { SplittwocolumnsInPlaceEdit as InPlaceEdit } from './SplittwocolumnsInPlaceEdit';
+
+const { apiFetch } = wp;
+const { addQueryArgs } = wp.url;
+
+export const SplittwocolumnsEditor = ({ attributes, setAttributes, isSelected }) => {
+  updateDeprecatedData(attributes, setAttributes, BLOCK_NAME, VERSION);
+  return (
+    <Fragment>
```

Comment:
This `Fragment` can probably be removed too... Since `renderEdit` has one, and `renderView` is only one `section`